tldw-frontend/extension/components/Option/Collections/ReadingList/ReadingItemDetail.tsx

```
@@ -344,16 +688,39 @@ export const ReadingItemDetail: React.FC<ReadingItemDetailProps> = ({
       children: (
         <div className="prose prose-sm dark:prose-invert max-w-none">
           {currentItem?.clean_html ? (
-            <div dangerouslySetInnerHTML={{ __html: currentItem.clean_html }} />
+            <div
+              ref={contentRef}
+              onMouseUp={captureSelection}
+              onKeyUp={captureSelection}
+              onTouchEnd={captureSelection}
+              dangerouslySetInnerHTML={{ __html: currentItem.clean_html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)